src/diet.ts

```diff
@@ -277,12 +279,17 @@ export function runDiet(): void {
   }
   useIfUnused($item`fancy chocolate car`, get("_chocolatesUsed") === 0, 2 * MPA);
 
-  const loveChocolateCount = Math.max(3 - Math.floor(20000 / MPA) - get("_loveChocolatesUsed"), 0);
-  const loveChocolateEat = Math.min(
-    loveChocolateCount,
-    itemAmount($item`LOV Extraterrestrial Chocolate`)
-  );
-  use(loveChocolateEat, $item`LOV Extraterrestrial Chocolate`);
+  while (get("_loveChocolatesUsed") < 3) {
+    const price = have($item`LOV Extraterrestrial Chocolate`) ? 15000 : 20000;
+    const value = clamp(3 - get("_loveChocolatesUsed"), 0, 3) * get("valueOfAdventure");
+    if (value < price) break;
+    if (!have($item`LOV Extraterrestrial Chocolate`)) {
+      Kmail.send("sellbot", `${$item`LOV Extraterrestrial Chocolate`.name} (1)`, undefined, 20000);
+      wait(11);
+      cliExecute("refresh inventory");
+    }
+    use($item`LOV Extraterrestrial Chocolate`);
```

Comment:
should put a break here if the kmail request doesn't return in time and you still don't have the chocolate